google/detectors/credentials/generic_weak_credential_detector/src/main/resources/detectors/credentials/genericweakcredentialdetector/data/service_default_credentials.textproto

```diff
@@ -55,6 +55,12 @@ service_default_credentials {
   default_usernames: "admin"
   default_passwords: "admin"
 }
+service_default_credentials {
```

Comment:
Could you add another default credential rstudio/rstudio? Creds declared here are being tested first, hopefully short-circuit the plugin more often.

Reply:
I've implemented the required change. However, since nmap doesn't recognize the service, the standard flow will be followed. This means that the credentials will be bruteforced in the regular way.